Britannia/localization/english/event_localization/BTWK_events_l_english.yml

```diff
@@ -25,26 +25,40 @@
+  BTWK_frontier_event.0203.a:0 "Prydain am byth!"  # Placeholder
+
 
   BTWK_frontier_event.0204:0 "Hadrian's Wall Restored"  # Every other player 
-  BTWK_frontier_event.0204_desc:0 "[restored_hadrians_wall_major_character.GetName)] has completed the restoration of Hadrian's Wall, returning dignity to the land in the wake of the Imperial withdrawal from Britain, leaving it to ruin . Many throughout Britannia have begun to praise him as '#italic Hadrianus#!' - the heir to the great Roman Emperor Hadrian. \n\nSome even say that completing such a momentous feat would make one worthy of the High Kingship... \n\n"
-  BTWK_frontier_event.0204_desc_contribution:0 "Your own contributions to the wall's renovation have not gone unnoticed. While the poems may only remember [GetScope(restored_hadrians_wall_major_character.GetFirstNameNoTooltipRegnal)] as the sole guiding hand of the effort, many of your subjects appreciate you in your own light."
+  BTWK_frontier_event.0204_desc_header:0 "[restored_hadrians_wall_major_character.GetName] has completed the restoration of Hadrian's Wall. In the wake of the Imperial witdrawal, Britain had been left in choas and the wall left to ruin. Now it once again proudly stretches from sea to sea. For this achievement, many throughout Britannia have begun to praise him as '#italic Hadrianus#!' - the heir to the great Roman Emperor Hadrian." 
+  BTWK_frontier_event.0204_desc_britannia_alive:0 "Of course, the reconstruction was only symbolic in nature, for Britannia's North extends well beyong the wall's limits."
+  BTWK_frontier_event.0204_desc_britannia_collapsed:0 "\n\nIn the wake of Britannia's demise, the return of the great fortification is a welcome sight to [restored_hadrians_wall_major_character.GetHerHis] people."
+
+  BTWK_frontier_event.0204_desc_not_high_king:0 "\n\nSome even say that completing such a momentous feat would make one worthy of the High Kingship..."
+  BTWK_frontier_event.0204_desc_high_king:0 "\n\nIf ever there was anyone to question the legitimacy of [restored_hadrians_wall_major_character.GetFirstNamePossessiveNoTooltip] rule over Britannia, now there are surely none."  # Use this for Imperials restoring the province of Britannia also?
+
+  BTWK_frontier_event.0204_desc_contribution:0 "\n\nYour own contributions to the wall's renovation have not gone unnoticed. While the poems may only remember [restored_hadrians_wall_major_character.GetFirstNameNoTooltipRegnal] as the sole guiding hand of the effort, many of your subjects appreciate you in your own light."
+
+  BTWK_frontier_event.0204_desc_caledonian:0 "\n\nThis may not be good news for our people, however. The heartlands of Britannia would be largely safe from assaults by land, now it has been returned. Without the threat of a flanking force to ruin their land, it could spell an impending invasion of our lands not seen since the days of Rome."
```

Comment:
The heartlands of Britannia would be largely safe from assaults by land, now it has been returned. -> Now that the wall has returned, the heartlands of Britannia are now largely safe from assaults over land.

Without the threat of a flanking force to ruin their land, it could spell an impending invasion of our lands not seen since the days of Rome. -> With Britannia secure, this could spell an impending threat to our independence not seen since the height of the empire.